config.yml

```diff
@@ -655,18 +725,78 @@ production_factories:
         material: COAL
         durability: 1
     inputs:
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 40
       Diamond:
         material: DIAMOND
-        amount: 50
+        amount: 160
+      Iron Ingot:
+        material: IRON_INGOT
+        amount: 200

```

Comment:
Curious why iron ingots is an ingredient for all top-tier factories? Only adds like 12d to the overall cost, seems superfluous?

Commentary aside, syntax seems fine so far.